ragna/source_storages/_chroma.py

```diff
@@ -96,19 +96,19 @@ def retrieve(
             include=["distances", "metadatas", "documents"],
         )
 
-        num_results = len(result["ids"])
+        num_results = len(result["ids"][0])
```

Comment:
This is the actual fix. For some reason, `result["ids"]` is a list, which contains nothing but another list, which ultimately holds the IDs we are after here.